.idea/.name

```diff
@@ -1 +0,0 @@
-video-kit-ffmpeg-android
```

Comment:
this file should be commited to the repo, please remove whole .idea catalogue and add it to the .gitignore.